spacelift/data_current_stack.go

```diff
@@ -22,7 +22,7 @@ func dataCurrentStackRead(d *schema.ResourceData, meta interface{}) error {
 	if err != nil {
 		// Don't care about validation errors, we don't actually validate those
 		// tokens, we only parse them.
-		if _, isValidation := err.(*jwt.ValidationError); !isValidation {
+		if !errors.Is(err, &jwt.UnverfiableTokenError{}) {
```

Comment:
😱 https://github.com/dgrijalva/jwt-go/issues/448

Once it's seen it can't be un-seen 😆